JSTutorial/hw_7/task_2.js

```diff
@@ -0,0 +1,299 @@
+ * addEnterprise("Название нового предприятия")
+ *
+ * 4. Написать функцию, которая будет добавлять отдел в предприятие. В качестве аргумента принимает id предприятия, в которое будет добавлен отдел и название отдела.
+ *
+ * Пример:
+ * addDepartment(1, "Название нового отдела")
+ *
+ * 5. Написать функцию для редактирования названия предприятия. Принимает в качестве аргумента id предприятия и новое имя предприятия.
+ *
+ * Пример:
+ * editEnterprise(1, "Новое название предприятия")
+ *
+ *
+ * 6. Написать функцию для редактирования названия отдела. Принимает в качестве аргумента id отдела и новое имя отдела.
+ *
+ * Пример:
+ * editDepartment(7, "Новое название отдела")
+ *
+ *
+ * 7. Написать функцию для удаления предприятия. В качестве аргумента принимает id предприятия.
+ *
+ * Пример:
+ * deleteEnterprise(1)
+ *
+ *
+ * 8. Написать функцию для удаления отдела. В качестве аргумента принимает id отдела. Удалить отдел можно только, если в нем нет сотрудников.
+ *
+ * Пример:
+ * deleteDepartment(3)
+ *
+ *
+ * 9. Написать функцию для переноса сотрудников между отделами одного предприятия. В качестве аргумента принимает два значения: id отдела, из которого будут переноситься сотрудники и id отдела, в который будут переноситься сотрудники).
+ *
+ * Пример:
+ * moveEmployees(2, 3)
+ */
+const enterprises = [
+    {
+        id: 1,
+        name: "Предприятие 1",
+        departments: [
+            {
+                id: 2,
+                name: "Отдел тестирования",
+                employees_count: 10,
+            },
+            {
+                id: 3,
+                name: "Отдел маркетинга",
+                employees_count: 20,
+            },
+            {
+                id: 4,
+                name: "Администрация",
+                employees_count: 15,
+            },
+        ]
+    },
+    {
+        id: 5,
+        name: "Предприятие 2",
+        departments: [
+            {
+                id: 6,
+                name: "Отдел разработки",
+                employees_count: 50,
+            },
+            {
+                id: 7,
+                name: "Отдел маркетинга",
+                employees_count: 20,
+            },
+            {
+                id: 8,
+                name: "Отдел охраны труда",
+                employees_count: 5,
+            },
+        ]
+    },
+    {
+        id: 9,
+        name: "Предприятие 3",
+        departments: [
+            {
+                id: 10,
+                name: "Отдел аналитики",
+                employees_count: 0,
+            },
+        ]
+    }
+]
+
+//1
+function countEmployees(departments) {
+    return departments.reduce(function(total, department) {
+        return total + department.employees_count;
+    }, 0);
+}
+console.log(countEmployees(enterprises[0].departments));
+const getCommonInfo = (company) => {
+    company.forEach(enterprise => {
+            console.log(`${enterprise.name} (${countEmployees(enterprise.departments)} сотрудников)`);
```

Comment:
```suggestion
            console.log(`${enterprise.name} (${countEmployees(enterprise.departments)} сотрудников)`);
```
Add handling for employees count value
`1 - сотрудник`
`2-4 - сотрудника`